WordPress/gutenberg
# Validation: Log debugging messages for invalid blocks

Closes #2837

This pull request seeks to add logging to the block validator to emit messages when an invalid block is encountered in a development environment.

![Validation](https://user-images.githubusercontent.com/1779930/32904950-96ce79d8-cac6-11e7-8dd8-8e173404b2e4.png)

__Testing instructions:__

Modify `post-content.js`, removing, adding, or changing attributes or content which would cause a block to become invalid (i.e. serialize differently on the next load). 

Load the editor and verify that debugging messages are logged in your Developer Tools console.
